tests/integration-tests/conftest.py

```diff
@@ -691,3 +693,24 @@ def architecture(request, instance, region):
 def key_name(request):
     """Return the EC2 key pair name to be used."""
     return request.config.getoption("key_name")
+
+
+@pytest.fixture()
+def pcluster_ami_without_standard_naming(region, os, architecture):
```

Comment:
Missing `version` parameter?

Reply:
No. version is passed in in the yielded funtion